### DISCUSSION TITLE
Interaction Prompt Aliasing

### DISCUSSION THREAD

**Discussion by wezside** [google/model-viewer#1918] [Q&A]
### Description

Aliasing problems while interaction prompt is running. Once we interact with the model the aliasing improves significantly  on IOS devices. This doesn't appear to happen on desktops or Android. It's as if a low-res version of the model is running during interaction prompt. 

### Version

- model-viewer: v1.2.1

### Browser Affected

- [ x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Attachments

[mv-aliaising-ios.zip](https://github.com/google/model-viewer/files/5238515/mv-aliaising-ios.zip)

**Comment by elalish** [comment]
This is actually a feature. You are correct that you are seeing a lower-res render while it is moving (not just the interaction prompt, but also auto-rotation or user interaction). This is done automatically to keep the frame rate up and avoid jank. You can disable it by setting `customElements.get("model-viewer").minimumRenderScale = 1`. The fact that you can't hit frame rate on iOS may indicate you want to optimize your model for faster rendering (reducing material count, vertex count, and texture sizes).

**Comment by wezside** [comment]
Thanks @elalish . Will give you recommendations a try.
